src/main/resources/templates/db_scripts/schema.json

```diff
@@ -23,7 +23,7 @@
     },
     {
       "tableName": "vouchers",
-      "fromModuleVersion": "mod-invoice-storage-1.0.0",
+      "fromModuleVersion": "mod-invoice-storage-3.0.0",
```

Comment:
I think we'll have to release 3.0.1  for this to be picked up by edelweiss, and perform  upgrades. Hence, the version must be the next release version: mod-invoice-storage-3.0.1

Reply:
Thank you!